src/anomalib/deploy/export.py

```diff
@@ -57,6 +57,12 @@ class CompressionType(str, Enum):
     Full integer post-training quantization (INT8)
     All weights and operations are quantized to INT8. Inference is done in INT8 precision.
     """
+    INT8_ACQ = "int8_acq"
+    """
+    Accuracy-control quantization (INT8)
+    Weights and operations are quantized to INT8, except those that would degrade quality of the model more than is
+    acceptable. Inference is done in a mixed precision.
+    """
```

Comment:
Would it be possible to add all of the above commented sections to the docstring. Something like this:
```python
class CompressionType(str, Enum):
    """Model compression type when exporting to OpenVINO.

    Attributes:
        FP16 (str): Weight compression (FP16). All weights are converted to FP16.
        INT8 (str): Weight compression (INT8). All weights are quantized to INT8,
            but are dequantized to floating point before inference.
        INT8_PTQ (str): Full integer post-training quantization (INT8).
            All weights and operations are quantized to INT8. Inference is done
            in INT8 precision.
        INT8_ACQ (str): Accuracy-control quantization (INT8). Weights and
            operations are quantized to INT8, except those that would degrade
            quality of the model more than is acceptable. Inference is done in
            a mixed precision.

    Examples:
        >>> from anomalib.deploy import CompressionType
        >>> CompressionType.INT8_PTQ
        'int8_ptq'
    """

    FP16 = "fp16"
    INT8 = "int8"
    INT8_PTQ = "int8_ptq"
```

This is especially useful to add these to the documentation.